CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ The format is based on [Keep a Changelog]
 
 ## [Unreleased]
 
+- Service operators can create and view a payroll run and provide a link to
+  Payroll operators (Cantium users) to download a file
```

Comment:
I'm wondering why you chose to capitalise "Payroll" here. We use the capitalisation "service operator" so I'd have thought it would also be "payroll operator".

Reply:
I think I just picked a style for wrtiing it out in commit messages and tried to stick to it, not wedded to it at all.